core/standard/recommendations/building-blocks/record-core/PER_json-media-types.adoc

```diff
@@ -0,0 +1,9 @@
+[[per_record-core_json-media-types]]
+[width="90%",cols="2,6a"]
+|===
+^|*Permission {counter:per-id}* |*/per/record-core/json-media-types*
+
+Record or catalog resources MAY be accompanied by a generic media type. Thus a JSON-encoded record or catalog resource MAY be accompanied by an `application/json` media type.
```

Comment:
What about application/geo+json for records?

Reply:
Added more text to clarify the permission.